scripts/variantstore/wdl/GvsExtractAvroFilesForHail.wdl

```diff
@@ -0,0 +1,208 @@
+version 1.0
+
+workflow GvsExtractAvroFilesForHail {
+    input {
+        String project_id
+        String dataset
+        String filter_set_name
+    }
+
+    call OutputPath { input: go = true }
+
+    call ExtractFromNonSuperpartitionedTables {
+        input:
+            project_id = project_id,
+            dataset = dataset,
+            filter_set_name = filter_set_name,
+            avro_sibling = OutputPath.out
+    }
+
+    call CountSamples {
+        input:
+            project_id = project_id,
+            dataset = dataset
+    }
+
+    # Superpartitions have max size 4000. The inner '- 1' is so the 4000th (and multiples of 4000) sample lands in the
+    # appropriate partition, the outer '+ 1' is to iterate over the correct number of partitions.
```

Comment:
This is the only thing that I'd want to see tested out (before the tie out) and I'm not sure of the best way to do this...maybe this can wait until we do this with 10k samples